drivers/wifi/nrf700x/Kconfig

```diff
@@ -351,3 +351,7 @@ endif # NETWORKING
 config WIFI_MGMT_SCAN_SSID_FILT_MAX
 	default 2
 endif
+
+config NRF_WIFI_BEAMFORMING
+	bool "Enable Wi-Fi beamforming"
+	default y
```

Comment:
Would you need some quick tradeoff explanation here?

Reply:
Added.